pages/contributors.js

```diff
@@ -0,0 +1,220 @@
+                                            github={item.github}
+                                            description={item.description}
+                                        ></Card>
+                                    </div>
+                                ))}
+                            </div>
+                        </div>
+                    ) : contributors ? (
+                        <div className="text-center">
+                            <Lottie options={defaultOptions} height={300} />
+                            <Text
+                                tag="h6"
+                                textSize="subheader"
+                                textColor="003e54bd"
+                                fontFamily="madetommy-bold"
+                            >
+                                No Contributors Right Now
+                            </Text>
+                        </div>
+                    ) : (
+                        <div className="text-center">
+                            <Spinner animation="border" role="status">
+                                <span className="sr-only">Loading...</span>
+                            </Spinner>
+                        </div>
+                    )}
+                </>
+            )}
+            {/* <div className="container py-4">
+                <div className="row no-gutters mt-2">
+                    {contributors.map((item, index) => (
+                        <div className="col-12 col-md-6 col-lg-3 p-3" key={item.handle_name}>
+                            <Card name={item.name} image={item.image} github={item.github}>
+                                {item.description}
+                            </Card>
+                        </div>
+                    ))}
+                </div>
+            </div> */}
+        </div>
+    )
+}
+
+function getContributors(contriFrontend, contriBackend) {
+    let common = contriFrontend.filter((x) => contriBackend.includes(x))
+    let frontend = contriFrontend.filter((x) => !common.includes(x))
+    let backend = contriBackend.filter((x) => !common.includes(x))
+
+    let arr = []
+    for (let x of common) {
+        arr.push({
+            handle_name: x.login,
+            name: x.login,
+            image: x.avatar_url,
+            github: x.html_url,
+            description: ["hackalog-frontend", "hackalog-backend"],
+        })
+    }
+    for (let x of frontend) {
+        arr.push({
+            handle_name: x.login,
+            name: x.login,
+            image: x.avatar_url,
+            github: x.html_url,
+            description: ["hackalog-frontend"],
+        })
+    }
+    for (let x of backend) {
+        arr.push({
+            handle_name: x.login,
+            name: x.login,
+            image: x.avatar_url,
+            github: x.html_url,
+            description: ["hackalog-backend"],
+        })
+    }
+    setContributors(arr)
+    console.log(contributors)
+}
```

Comment:
Remove this unused block of code

Reply:
yes, sorry i forgot to remove this
I ll change it